lib/services/ios-project-service.ts

```diff
@@ -5,6 +5,7 @@ import * as path from "path";
 import * as shell from "shelljs";
 import * as util from "util";
 import * as os from "os";
+import * as semver from "semver";
```

Comment:
Are you sure this work? semver is exported as 
`
declare module "semver" {
	export = SemVerModule;
}
`
and I expect that only imports with require work.